modules/bpi/bpi.push.inc

```diff
@@ -63,7 +63,7 @@ function bpi_http_push_action_form($nid) {
     ),
     '#description' => t('You should have permission to publish the images before selecting this option.'),
     '#default_value' => '_none',
-    '#element_validate' => array('bpi_push_with_images_validate'),
+    '#required' => TRUE,
```

Comment:
It seems as if this should be handled by setting ` '#required' => TRUE` and then defining `#empty_option` for the element? Then we could drop the `_none` values and the custom validation?.